ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -422,14 +426,24 @@ fun TaskEntityAndRelations.toModelObject(): Task {
     multipleChoice = multipleChoiceEntities[0].toModelObject(optionEntities)
   }
 
+  var condition: Condition? = null
+
+  if (conditionEntityAndRelations.isNotEmpty()) {
+    if (conditionEntityAndRelations.size > 1) {
+      Timber.e("More than 1 condition found for task")
```

Comment:
Maybe let's include the task itself in the log statement for debug info.

Reply:
Done.